### PR TITLE
Kernel Constructor Cleanup -- issue 5030

### PR DESCRIPTION
**Pull request by GuoxinYin**
This is some cleanup trying to fix issue #5030
@gf712 Could you please help review it? Thanks!

### CODE REVIEW

File: src/shogun/kernel/GaussianShortRealKernel.h [hunk 1]
```diff
@@ -70,9 +70,7 @@ class GaussianShortRealKernel: public DotKernel
 		 * @return name GaussianShortReal
 		 */
 		const char* get_name() const override { return "GaussianShortRealKernel"; }
-		/** register the parameters
-		 */
-		void register_params() override;
+
```
Comment (gf712) [inline src/shogun/kernel/GaussianShortRealKernel.h:73]: remove whitespace

File: src/shogun/kernel/HistogramIntersectionKernel.h [hunk 2]
```diff
@@ -58,9 +58,7 @@ class HistogramIntersectionKernel: public DotKernel
 		 * @return if initializing was successful
 		 */
 		bool init(std::shared_ptr<Features> l, std::shared_ptr<Features> r) override;
-		/* register the parameters
-		 */
-		void register_params() override;
+
```
Comment (gf712) [inline src/shogun/kernel/HistogramIntersectionKernel.h:61]: remove whitespace

File: src/shogun/kernel/SphericalKernel.h [hunk 3]
```diff
@@ -102,8 +102,7 @@ class SphericalKernel: public Kernel
 	~SphericalKernel() override;
 
 private:
-	/** register parameters */
-	void register_params() override;
+
```
Comment (gf712) [inline src/shogun/kernel/SphericalKernel.h:105]: remove whitespace

File: src/shogun/kernel/WeightedDegreeRBFKernel.h [hunk 4]
```diff
@@ -134,8 +134,7 @@ class WeightedDegreeRBFKernel: public DotKernel
 		float64_t* weights;
 
 	private:
-		/** register parameters */
-		void register_params() override;
+
```
Comment (gf712) [inline src/shogun/kernel/WeightedDegreeRBFKernel.h:137]: remove whitespace

**Review by gf712 — CHANGES_REQUESTED** [review]
lgtm, just need to remove some whitespaces. Thanks!

**Comment by GuoxinYin** [comment]
Hi @gf712, I have finished removing the whitespaces. Please help review it. Thanks!

**Comment by gf712** [comment]
Thanks for this!

**Pull request merged**